File: fizz-buzz.go
package main
import (
	"fmt"
	"strconv"
)

func fizzbuzz(i int) string {
	if i % 3 == 0 && i % 5 == 0 { return "Fizz Buzz" }
	if i % 3 == 0 { return "Fizz" }
	if i % 5 == 0 { return "Buzz" }
	return strconv.Itoa(i)
}

func main() {
	var n int
	fmt.Scan(&n)
	for i := 1; i <= n; i++ {
		fmt.Println(fizzbuzz(i))
	}
}

File: fizz-buzz.rb
def fizzbuzz(i):
	if i % 3 == 0 and i % 5 == 0
		return "Fizz Buzz"
	elsif i % 3 == 0
		return "Fizz"
	elsif i % 5 == 0
		return "Buzz"
	else
		return i
	end
end

for i in 1..gets.to_i
	puts fizzbuzz(i)
end
